hw/ip/clkmgr/dv/cov/clkmgr_manual_cov_unr_excl.el

```diff
@@ -0,0 +1,49 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This excludes toggle coverage for output ports driven by constant and inputs
+// tied to constants.
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Aug 11 12:43:08 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2617278896 3012009448"
+INSTANCE: tb.dut
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.aon_infra "logic cg_en_o.aon_infra[3:0]"
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.aon_powerup "logic cg_en_o.aon_powerup[3:0]"
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.io_div2_powerup "logic cg_en_o.io_div2_powerup[3:0]"
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.aon_timers "logic cg_en_o.aon_timers[3:0]"
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.aon_secure "logic cg_en_o.aon_secure[3:0]"
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.io_div4_powerup "logic cg_en_o.io_div4_powerup[3:0]"
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.usb_powerup "logic cg_en_o.usb_powerup[3:0]"
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.main_powerup "logic cg_en_o.main_powerup[3:0]"
+ANNOTATION: "[UNR] This is driven by a constant."
+Toggle cg_en_o.io_powerup "logic cg_en_o.io_powerup[3:0]"
+CHECKSUM: "2301929872 1660332954"
+INSTANCE: tb.dut.u_clk_main_aes_trans.u_idle_cnt
+ANNOTATION: "[UNR] Input tied to a constant."
```

Comment:
should we tag this UNSUPPORTED or LOWRISC? These are not really unreachable, right?

Reply:
I don't understand this comment: the parent module ties these to constants, so the input port won't toggle, meaning toggle transitions are unreachable. UNTOGGLEABLE is too much of a mouthful. Why LOWRISC? UNSUPPORTED implies the tool doesn't support this case, but we don't need to mention tool shortcomings.

Reply:
I feel like this should have been discovered by the UNR tool.

Reply:
I thought these were dut input controlled by TB, but they're prim_count inputs. 
Then, UNR tool should catch them, like Sri said.

Reply:
These could be flagged to the VCS support team, but we should not wait for them to diagnose and fix it.